common/src/main/cpp/include/extensions/openxr_fb_render_model.h

```diff
@@ -0,0 +1,69 @@
+/**************************************************************************/
+/*  openxr_fb_render_model.h                                              */
```

Comment:
Similar comment as for `openxr_fb_render_model.cpp`; let's move this header under a new `include/classes` subdirectory: `common/src/main/cpp/include/classes`